src/main/kotlin/Calculator.kt

```diff
@@ -0,0 +1,46 @@
+class Calculator {
+    private val add = "+"
+    private val minus = "-"
+    private val times = "*"
+    private val divide = "/"
+
+    fun calculate(formula: String): Double {
+        if (formula.isBlank()) {
+            throw IllegalArgumentException("값을 입력해 주세요")
+        }
```

Comment:
`require()`는 값을 만족하지 않으면 `IllegalArgumentException`을 발생시킵니다.

```suggestion
        require(formula.isNotBlank()) { "값을 입력해 주세요" }
```

Reply:
적용하였습니다.